themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
+	width: 1em;
+	height: 1em;
+	vertical-align:middle;
```

Comment:
vertical-align is already inhereted from:  label svg
```
button svg, button img, label svg, label img {
    vertical-align: middle;
}
```

it's the input box, that needs a vertical-alignment.